security/org-account-assume/main.tf

```diff
@@ -101,56 +91,3 @@ resource "aws_iam_role_policy" "prime-admin" {
   provider = aws.workload
 }
 
-# Create cloud-engineer role for the workload account
-resource "aws_iam_role" "cloudengineer_role" {
-  name                 = var.cloudengineer_iam_role_name
```

Comment:
Is this var used anywhere else?

Reply:
Nope, spotted and tested removal of these before lunch but hadn't pushed. Good that you've got a keen eye!